src/test/java/org/amoseman/InitTestConfiguration.java

```diff
@@ -16,7 +16,11 @@ public static void init(String configurationLocation, String databaseLocation, S
             String contents =   "database-url: " + databaseLocation + "\n" +
                     "admin-username: " + username + "\n" +
                     "admin-password: " + password + "\n" +
```

Comment:
You should avoid writing credentials anywhere on the filesystem.  Assume it is insecure.